core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -332,7 +330,20 @@ public boolean exists(Item f) throws FileSystemException {
 			if (findResults.getTotalCount() == 0) {
 				return FileSystemUtils.getDirectoryStream(null);
 			} else {
-				return FileSystemUtils.getDirectoryStream(findResults.getItems().iterator());
+				Iterator<Item> itemIterator = findResults.getItems().iterator();
+				return FileSystemUtils.getDirectoryStream(new Iterator<EmailMessage>() {
```

Comment:
Ik snap niet waarom je dit hier doet

Reply:
must cast <Items> to <EmailMessage> separately, cannot cast Iterator<Item> to Iterator<EmailMessage>